src/data/battle-anims.ts

```diff
@@ -515,13 +514,30 @@ export function initMoveAnim(scene: BattleScene, move: Moves): Promise<void> {
             } else {
               resolve();
             }
+          })
+          .catch(error => {
+            useDefaultAnim(move, defaultMoveAnim, error);
+            return resolve();
```

Comment:
We should probably print at least a log, better yet a warning here
```suggestion
          .catch(error => {
            console.warn(`No move animation found for ${Utils.animationFileName(move)}! Falling back to default.`);
            useDefaultAnim(move, defaultMoveAnim, error);
            return resolve();
```

Reply:
@flx-sta hmm the useDefaultAnim function already logs an error to the console

Reply:
renamed method to make that behavior more clear